tests/integration/cloud/rest_contracts/test_expectation_suites.py

```diff
@@ -366,15 +339,15 @@ def test_put_non_existent_expectation_suite(
 
 
 @pytest.mark.cloud
-@pytest.mark.skip(reason="unexpected 401 instead of 204 in CI only")
+# @pytest.mark.skip(reason="unexpected 401 instead of 204 in CI only")
```

Comment:
Should we delete this instead?

Reply:
ah yeah definitely! i was curious to see if it passed, and i guess it does 😄